pony_gp.py

```diff
@@ -454,13 +453,9 @@ def coevaluate_fitness(population, adversaries, param, cache):
         exemplars = {"cases": [], "targets": []}
         adversary_fitness = []
         for case in param["fitness_cases"]:
-            case_p = evaluate(adversary["genome"], case)
-            # TODO transform variables to correct dimensions
-            case_p = [case_p - i for i in range(len(param["variables"]))]
-            target = evaluate(param["system_function"], case_p)
-            exemplars["cases"].append(case_p)
-            exemplars["targets"].append(target)
-            
+            exemplars["cases"].append(case[0:-1])
```

Comment:
The same `fitness_cases` are used all the time and the `adversary` is never evaluated. I.e. it seems like the adversary will always have the same fitness